activesupport/CHANGELOG.md

```diff
@@ -67,6 +67,21 @@
 
     *Andrew White*
 
+*   Deprecates Rails custom `Enumerable#sum` and `Array#sum` in favor of Ruby's native implementation which
+    is considerably faster.
+
+    Ruby requires an initializer for non-numeric type as per examples below:
```

Comment:
@pixeltrix Did you test it on `Money` objects? It looks like Ruby natively supports `Money` objects without initial arguments.
```ruby
require 'money'
[Money.new(1000), Money.new(1000), Money.new(1000)].sum
=> #<Money fractional:3000 currency:USD>

Money.new(1000).is_a? Numeric
 => false
```
Money object is not a `Numeric` but it works natively in Ruby so I wonder why you would raise a deprecation warning. In my project we have hundreds of occurrences  of `sum` of money objects so I wonder why I would need to change it. It would be a very time consuming and redundant effort.